artifactory/services/users.go

```diff
@@ -0,0 +1,139 @@
+package services
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/jfrog/jfrog-client-go/auth"
+	"github.com/jfrog/jfrog-client-go/http/jfroghttpclient"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/httputils"
+)
+
+type UsersParams struct {
```

Comment:
UsersParams --> UserParams
Let's rename the relevant methods as well, like ```NewUsersParams``` (should be renamed to ```UserParams```.